pkg/sidecar/sidecar_test.go

```diff
@@ -0,0 +1,139 @@
+package sidecar
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"github.com/testground/sdk-go/runtime"
+	"github.com/testground/sdk-go/sync"
+	"math/rand"
+	"strconv"
+	"testing"
+	"time"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+const (
```

Comment:
```suggestion
const (
	// dummy network configuration params that we request and later verify that sidecar has applied.
```

It is not immediately clear what these values are for, so maybe worth adding a comment.